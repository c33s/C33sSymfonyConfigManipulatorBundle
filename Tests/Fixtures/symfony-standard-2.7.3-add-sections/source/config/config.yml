imports:
    - { resource: parameters.yml }
    - { resource: security.yml }
    - { resource: services.yml }
    - { resource: config/assetic.yml }
    - { resource: config/doctrine.yml }
    - { resource: config/framework.yml }
    - { resource: config/parameters.yml }
    - { resource: config/swiftmailer.yml }
    - { resource: config/twig.yml }

# this is a newly added section that should be moved into a sub directory
foo:
    bar: baz
