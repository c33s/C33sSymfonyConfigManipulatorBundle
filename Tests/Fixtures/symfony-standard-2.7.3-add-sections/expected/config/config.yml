imports:
    - { resource: parameters.yml }
    - { resource: security.yml }
    - { resource: services.yml }
    - { resource: config/assetic.yml }
    - { resource: config/doctrine.yml }
    - { resource: config/foo.yml }
    - { resource: config/framework.yml }
    - { resource: config/parameters.yml }
    - { resource: config/swiftmailer.yml }
    - { resource: config/twig.yml }
